jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/common/Chip.java

```diff
@@ -76,13 +66,13 @@ public boolean selected() {
         return hasClass("v-chip--active");
```

Comment:
Comment and check are equal, but method name is about selection, should be renamed